astropy/units/equivalencies.py

```diff
@@ -162,13 +166,15 @@ def spectral_density(wav, factor=None):
     wav : `~astropy.units.Quantity`
         `~astropy.units.Quantity` associated with values being converted
         (e.g., wavelength or frequency).
+    factor : array_like
+        If ``wav`` is a |Unit| instead of a |Quantity| then ``factor``
+        is the value ``wav`` will be multiplied with to convert it to
+        a |Quantity|.
```

Comment:
```suggestion
    factor : array_like
        If `wav` is a |Unit| instead of a |Quantity| then `factor`
        is the value `wav` will be multiplied with to convert it to
        a |Quantity|.
```